src/main/java/com/example/huiswerktechiteasycontrollerspringboot/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,61 @@
+package com.example.huiswerktechiteasycontrollerspringboot.controllers;
+
+import com.example.huiswerktechiteasycontrollerspringboot.exceptions.StringTooLongException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionsController {
+    public static ArrayList<String> televisionDataBase = new ArrayList<>(Arrays.asList("Philips 200", "Samsung 531", "Nikkei 1", "LG OLED TV", "Sony Bravia TV", "TCL Roku TV"));
+
+    @GetMapping()
+    public ResponseEntity<Object> getAllTelevisions() {
+        return new ResponseEntity<>(televisionDataBase, HttpStatus.OK);
+    }
+
+    // Gebruik voor id int, omdat ik de id als index gebruik. Begreep dat long netter is voor id.
+    @GetMapping("/{id}")
+    public ResponseEntity<Object> getOneTelevision(@PathVariable int id) throws IndexOutOfBoundsException {
+        if (id < 0 || id >= televisionDataBase.size()) {
+            throw new IndexOutOfBoundsException("The TV with id number " + id + " does not exist");
+        }
+        return new ResponseEntity<>(televisionDataBase.get(id), HttpStatus.OK);
+    }
+
+    @PostMapping("/add")
+    public ResponseEntity<Object> addTelevision(@RequestBody String television) throws StringTooLongException {
+        if (television.length() > 20) {
+            throw new StringTooLongException("The television name is too long. The name of a TV can have a maximum of 20 characters.");
```

Comment:
je zou voor betere feedback nog kunnen zeggen om welke string het precies gaat die te lang is door de string zelf ook mee te geven.